valkey/values.yaml

```diff
@@ -1,9 +1,17 @@
+global:
+  # The global image registry (this will override the registry of all container images defined in this chart)
+  imageRegistry: ""
+  # The global image pull secrets (list of secret names)
+  imagePullSecrets: []
+
 # Number of Valkey pods to run
 replicaCount: 1
 
 image:
-  # Valkey Docker image to use
-  repository: docker.io/valkey/valkey
+  # Image registry
+  registry: "docker.io"
```

Comment:
Defining image.registry (and metrics.image.registry) is a potential breaking change.